pallets/subtensor/src/weights.rs

```diff
@@ -491,4 +547,47 @@ impl<T: Config> Pallet<T> {
 
         false
     }
+
+    /// Get the last processed nonce for a given netuid and account
+    ///
+    /// # Arguments
+    ///
+    /// * `netuid` - The network ID
+    /// * `account` - The account ID
+    ///
+    /// # Returns
+    ///
+    /// Returns the last processed nonce for the given netuid and account, or 0 if not found
+    ///
+    /// # Note
+    ///
+    /// This function is used to retrieve the last nonce that was processed for a specific account
+    /// in a specific subnet. It's crucial for maintaining the order of transactions and
+    /// preventing replay attacks.
+    pub fn get_last_processed_nonce(netuid: u16, account: &T::AccountId) -> u64 {
+        LastProcessedNonce::<T>::get(netuid, account)
+        // ^ If no nonce is found, we return 0 as the default value
+    }
```

Comment:
I think we should just use `LastProcessedNonce::<T>::get(netuid, account)` directly in-line in our code, instead of adding all this boilerplate

Reply:
We have not decided on whether or not to start accessing storage values directly in code , and this follows the current pattern of creating getters and setters for storage values.